twirl

```diff
@@ -546,7 +546,7 @@ gem_install_or_update() {
 }
 
 install_brews() {
-    if test ! $(brew list | grep $brew); then
+    if test ! $(brew list | grep -x $brew); then
```

Comment:
More info about this option:
[![image](https://user-images.githubusercontent.com/385232/97759355-c8529900-1af8-11eb-9e3e-4209bbac6394.png)](http://linuxcommand.org/lc3_man_pages/grep1.html)